ChangeLog

```diff
@@ -1,3 +1,8 @@
+2024-07-12  Vedant Tewari <vedanttewari541@gmail.com>
```

Comment:
formatting: please have ChangeLog entries start with an empty line, this helps to reduce merge problems